Realm/Tests/DictionaryPropertyTests.m

```diff
@@ -587,6 +587,25 @@ - (void)testObjectAggregate {
     test();
 }
 
+- (void)testPrimitiveAggregate {
+    RLMRealm *realm = [RLMRealm defaultRealm];
+
+    __block DictionaryPropertyObject *obj = [DictionaryPropertyObject new];
+    obj.intDictionary[@"age"] = @10;
+    obj.intDictionary[@"weight"] = @200;
+    [obj.intDictionary minOfProperty:@"self"];
+
+    [realm transactionWithBlock:^{
+        DictionaryPropertyObject *managed = [DictionaryPropertyObject createInRealm:realm withValue:@[]];
+        managed.intDictionary[@"age"] = @20;
+        managed.intDictionary[@"weight"] = @400;
+        obj = managed;
+    }];
+
+    [obj.intDictionary minOfProperty:@"self"];
```

Comment:
Is this left over code from when you were experimenting with the aggregates?

Reply:
Yes.